tools/perf/components/browser_type.py

```diff
@@ -19,11 +19,57 @@
 from zipfile import ZipFile
 from distutils.dir_util import copy_tree
 
-from components import path_util
+import components.path_util as path_util
 from components.perf_test_utils import GetProcessOutput
 
 
-def _GetBraveDownloadUrl(tag: str, binary: str) -> str:
+class _BaseVersion:
+  _version: List[int]
+
+  def equals(self, a: _BaseVersion) -> bool:
+    return self._version == a.version()
+
+  def less(self, a: _BaseVersion) -> bool:
```

Comment:
why not `def __eq__(self, other):` and `def __lt__(self, other):`?

Reply:
Fixed